### PR TITLE
Table Jointure 

### PR DESCRIPTION
**Pull request by ridabensalem**
In this milestone of the vet-clinic project we have added the following requirements : 
- Create a table named owners 
- Create a table named species
- Modify animal's table
- Insert  new data into the owners table 
- Insert new  data into the species  table 
- Modify inserted animals so it includes the species_id value
- Modify inserted animals to include owner information (owner_id)
- Write queries (using JOIN) to answer the following questions:
- [ ] What animals belong to Melody Pond?
- [ ] List of all animals that are Pokemon (their type is Pokemon).
- [ ] List all owners and their animals, remember to include those that don't own any animal.
- [ ] How many animals are there per species?
- [ ] List all Digimon owned by Jennifer Orwell.
- [ ] List all animals owned by Dean Winchester that haven't tried to escape.
- [ ] Who owns the most animals?
- Add Screenshots to the README file 
##Screenshots of Jointures table 
Jointure_1
Jointure_2
Jointure_3
Jointure_4
Jointure_5
Jointure_6
Jointure_7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +83,14 @@
...
 <img src="./screenshots/QU12.png"/>
 <img src="./screenshots/QU13.png"/>
+##Screenshots of Jointures table 
+<img src="./screenshots/Jointure_1.png"/>
+<img src="./screenshots/Jointure_2.png"/>
+<img src="./screenshots/Jointure_3.png"/>
+<img src="./screenshots/Jointure_4.png"/>
+<img src="./screenshots/Jointure_5.png"/>
+<img src="./screenshots/Jointure_6.png"/>
+<img src="./screenshots/Jointure_7.png"/>
```
Comment (Ibizugbe) [inline README.md]: I can see you added the images of your queries to your readme file, but let's remember that the requirement is that the images are added to your PR description😉.
image
kindly edit your PR description and add all the necessary images showing each queries and the result in the terminal😉.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @ridabensalem 👋👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Animals table have been updated as required👍.
- [x] A new table for spacies and owners have been created👍.
- [x] speciies and owners table have been populated with the required data👍
- [x] Queries have been written as required👍.


## Required Changes ♻️

- [ ] Your MIT License link is broken, kindly do well to fix it as this gives your project a less professional look😉.
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ridabensalem** [comment]
Thank you for your feedback The MIT.md file has been added :)

File: README.md [hunk 2]
```diff
@@ -83,6 +83,14 @@
...
```
Comment (emmanuellekamwa) [inline README.md]: - [ ] As said by the previous reviewer, the screenshot of your query results should be inside the PR and not in the README file. Please kindly do so.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔⛔**
Hello @ridabensalem 👋
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Good commit messages.
- Correct git-flow.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @ridabensalem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**